### ISSUE TITLE
support generic tags (e.g. zone) for TrafficPermission source

### ISSUE THREAD

**Issue by bdecoste** [kumahq/kuma#1782]
### Summary

I'd like to use source tags for TrafficPermissions:

```
apiVersion: kuma.io/v1alpha1
kind: TrafficPermission
mesh: default
metadata:
  name: zone-traffic
spec:
  sources:
    - match:
        kuma.io/service: '*'
        kuma.io/zone: 'universal'
  destinations:
    - match:
        kuma.io/service: '*'
        kuma.io/zone: 'universal'
```

### Steps To Reproduce

```
$ kubectl -n kong-mesh-system apply -f zone-trafficpermission.yaml 
The TrafficPermission "zone-traffic" is invalid: 
* spec.sources[0].match: must consist of exactly one tag "kuma.io/service"
* spec.sources[0].match["kuma.io/zone"]: tag "kuma.io/zone" is not allowed
```

**Comment by jewertow** [comment]
It was implemented in #1902.

**Comment by jakubdyszkiewicz** [comment]
@jewertow is right, closing the issue.
